src/database/helpers/helpers.factory.js

```diff
@@ -0,0 +1,80 @@
+module.exports = ({
+    State,
+    Secret,
+    Notification,
+    Log
+}) => {
+    const getState = (data) => State.find(data); 
+    return {
+        saveSecrets: ({
+            team_id,
+            access_token,
+            bot_user_id,
+            bot_access_token
+        }) => {
```

Comment:
using the `function` keyword for a function as large as this makes it more readable